notifications_api_common/kanalen.py

```diff
@@ -39,7 +47,28 @@ def __repr__(self):
             self.main_resource,
         )
 
-    def get_kenmerken(self, obj: Model, data: Dict = None) -> Dict:
+    @staticmethod
+    def get_field(model: Model, field: str) -> Field:
+        """
+        Function to retrieve a field from a Model
+        """
+        return model._meta.get_field(field)
+
+    def get_help_text(self, field: Field, kenmerk: str) -> str:
+        """
+        Retrieve the help_text for a kenmerk, pulled from the model field by default,
+        but can be overridden by setting extra_kwargs on `Kanaal.__init__`
+        """
+        if help_text := self.extra_kwargs.get(kenmerk, {}).get("help_text"):
```

Comment:
I think this functionality should be tested

Reply:
Also does this mean i expects:
 extra_kwargs = {"kenmerk_1": {"help_text": "help text 2"}, "kenmerk_2": {"help_text": "help text 2"}}

Reply:
added a test for it